### ISSUE TITLE
sample interval is not set when subs are defined in a file

### ISSUE THREAD

**Issue by hellt** [karimra/gnmic#86]
subscribe interval is not set when the subs is defined in a file and not from CLI:

example of a request sent via file:
```
gnmic 2020/07/11 21:38:04.395276 sending gNMI SubscribeRequest: subscribe='subscribe:{prefix:{}  subscription:{path:{elem:{name:"state"}  elem:{name:"port"  key:{key:"port-id"  value:"1/1/c1/1"}}  elem:{name:"statistics"}  elem:{name:"out-octets"}}}  qos:{marking:20}}', mode='STREAM', encoding='JSON', to 0.tcp.eu.ngrok.io:x
```

the same sub request made via CLI:
```
gnmic 2020/07/11 21:37:08.030590 sending gNMI SubscribeRequest: subscribe='subscribe:{prefix:{}  subscription:{path:{elem:{name:"state"}  elem:{name:"port"  key:{key:"port-id"  value:"1/1/c1/1"}}  elem:{name:"statistics"}  elem:{name:"out-octets"}}  sample_interval:10000000000}  qos:{marking:20}}', mode='STREAM', encoding='JSON', to 0.tcp.eu.ngrok.io:x
```

**Comment by karimra** [comment]
I left it to zero to comply with this statement:
If the sample_interval is set to 0, the target MUST create the subscription and send the data with the lowest interval possible for the target
Meaning that the default for a file subscription would be zero.
Now the question becomes: do we want to align the cli flag def value or the file subscription?

**Comment by hellt** [comment]
Agreed that default sample-interval should be 0. Closed in favor of #88
